packages/relay-kit/src/packs/gelato/GelatoRelayPack.ts

```diff
@@ -69,7 +70,7 @@ export class GelatoRelayPack implements RelayPack {
     onlyCalls = false,
     options = {}
   }: CreateTransactionProps): Promise<SafeTransaction> {
-    const { gasLimit, isSponsored = false } = options
+    const { isSponsored = false } = options
     const nonce = await safe.getNonce()
     const chainId = await safe.getChainId()
```

Comment:
We also execute this two calls in the helper functions when transaction is not sponsored, so either we set this inside the if (isSponsored) or we pass them as parameter to the HandlePayment and WithTransfer functions